doc/source/overview/high_level.md

```diff
@@ -0,0 +1,508 @@
+# Practical Overview of Explainability
+
+While the applications of machine learning are impressive many models provide predictions that are hard to interpret or reason about. This limits their use in many cases as often we want to know why and not just what a model made a prediction. Alarming predictions are rarely taken at face value and typically warrant further analysis. Indeed, in some
+cases explaining the choices that a model makes could even become a potential [legal requirement](https://arxiv.org/pdf/1711.00399.pdf). The following is a non-rigorous and practical overview of explainability and the methods alibi provide.
```

Comment:
alibi --> Alibi (globally please)
provide --> provides

Reply:
Would avoid linking directly to pdf (just the arxiv abstract is fine) and also not use "non-rigorous" which sounds more negative than it should. Just a practical overview would be fine.